MainWindow.xaml

```diff
@@ -22,18 +23,18 @@
         <Image Source="/Assets/OFGB.png" SnapsToDevicePixels="True" Panel.ZIndex="1" Margin="12,6,0,0" Height="16" VerticalAlignment="Top" HorizontalAlignment="Left" />
         <Button Content="&#xe8bb;" FontSize="10" Panel.ZIndex="1" Width="40" Click="Close" Height="30" WindowChrome.IsHitTestVisibleInChrome="True" />
 
```

Comment:
A dropdown with the available languages / flags would be neat for the user to manually pick from?